guides/v2.1/frontend-dev-guide/css-topics/css_debug.md

```diff
@@ -13,6 +13,8 @@ The topic describes how to install, configure and use [Grunt JavaScript task run
 
 Make sure that you [set]({{ page.baseurl }}/config-guide/cli/config-cli-subcommands-mode.html) your Magento application to the developer or default [mode]({{ page.baseurl }}/config-guide/bootstrap/magento-modes.html).
 
+Also Less compilation modes whould be **Server-side Less compilation** [mode]({{ page.baseurl }}/css-topics/css-preprocess.html).
```

Comment:
Do users need to explicitly set the Less compilation mode to be Server-side Less compilation as a prerequisite? If that's what is already set by default, then this is not necessarily a prerequisite, but rather an FYI. If that's the case, we can change this to a note/tip.

Reply:
Hello @erikmarr,
Great, we can add it in note/tip, The reason behind to add this is that, whenever someone changed it from Server to client side so they should know about that configuration.

Reply:
Ok so rather than have a new line or a note here, what about adding something like this just after the first prereq sentence:
"The default mode sets the Less compilation mode to **Server-side Less compilation**.